src/tests/run.rs

```diff
@@ -142,6 +143,26 @@ fn connect_wrong_serial_number(_model: nitrokey::Model) {
   );
 }
 
+#[test_device]
+fn connect_usb_path(_model: nitrokey::Model) -> anyhow::Result<()> {
+  let devices = nitrokey::list_devices()?;
+  for path in devices.iter().map(|d| &d.path) {
+    let res = Nitrocli::new().handle(&["status", &format!("--usb-path={}", path)]);
+    assert!(res.is_ok());
```

Comment:
Could we do the same you did with the serial number where you check the serial number in the status output against the expected one taken from the device we connect to?

Reply:
The status currently does not include the device path because we use the `with_device` function that does not expose the `DeviceInfo` with the path.  I’m currently preparing a patch that makes the `status` command print the status for all attached devices.  We will have to call `list` directly, so we will be able to include the device path in the output.  But for now, I think it is the easiest solution to just make sure that the command is successful.

Reply:
What I mean is (roughly):
```rust
  for path in devices.iter().map(|d| &d.path) {
    let serial_number = d.serial_number.unwrap();
    let output = Nitrocli::new().handle(&["status", &format!("--usb-path={}", path)])?;
    assert!(res.contains(&format!("serial number:     {}\n", sn)));
```

So instead of verifying the USB path directly just take the route over the serial number. Still won't provide better coverage for storage devices, but I think it's fine. Once (or perhaps: if) storage devices report their serial number properly it will just work.

Reply:
Okay, done.